### ISSUE TITLE
 uninitialized constat Nem::Mixin::MosaicAttachable::Forwardable (NameError)

### ISSUE THREAD

**Issue by lengshuiyulangcn** [44uk/nem-ruby#1]
When I tried the example, I had the following error messages.

```
➜  try-nem-ruby ruby multisig_aggregate_modification.rb
/Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin/mosaic_attachable.rb:4:in `<module:MosaicAttachable>': uninitialized consta$
t Nem::Mixin::MosaicAttachable::Forwardable (NameError)
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin/mosaic_attachable.rb:3:in `<module:Mixin>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin/mosaic_attachable.rb:2:in `<module:Nem>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin/mosaic_attachable.rb:1:in `<top (required)>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin.rb:2:in `block in <top (required)>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin.rb:2:in `each'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem/mixin.rb:2:in `<top (required)>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/nem-ruby-0.0.7/lib/nem.rb:7:in `<top (required)>'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
        from /Users/gyorou/.anyenv/envs/rbenv/versions/2.4.3/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:39:in `require'
        from multisig_aggregate_modification.rb:2:in `<main>'
```

The reason seems we need to `require 'forwardable'` before use it since it's not in the ruby-core  library but belongs to the default library.

**Comment by 44uk** [comment]
Thank you for your cooperation! 👍

**Comment by 44uk** [comment]
Sorry, the gem this problem solved has not been released.

**Comment by 44uk** [comment]
This problem has been resolved 0.0.8
